.github/workflows/publish_to_pypi.yml

```diff
@@ -3,26 +3,26 @@ name: Publish Python 🐍 distributions 📦 to PyPI
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 jobs:
   build-n-publish:
     name: Build and publish Python 🐍 distributions 📦 to PyPI
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.7
-    - name: Install dependencies
-      run: pip install --upgrade setuptools setuptools_scm wheel twine
-    - name: Build package
-      run: python setup.py sdist bdist_wheel
-    - name: Upload deploy
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.PYPI_SECRET }}
-      run: python -m twine upload dist/*
+      - uses: actions/checkout@v2
```

Comment:
Not that it is a big of a deal, but why did you change indentation?

Reply:
I will blame vscode, which was insistent on fixing this indentation..